### DISCUSSION TITLE
我制作了一个Magpie9的win7适配版

### DISCUSSION THREAD

**Discussion by HIllya51** [Blinue/Magpie#553] [Show and tell]
https://github.com/HIllya51/Magpie9_win7
https://github.com/HIllya51/Magpie9_win7/releases/download/ok/Magpie9_win7.zip
😁（仅支持GDI，依赖.net48）

![Clip_20230407_193118](https://user-images.githubusercontent.com/101191390/230606246-cefae2ed-5568-443c-8ef8-ddc2bf4617d5.png)
![Clip_20230407_193104](https://user-images.githubusercontent.com/101191390/230606260-1688fd4c-dfc6-4360-b472-fcf3fee97202.png)

**Comment by Blinue** [comment]
太棒了。Graphics Capture 要求 Win10 1903，Desktop Duplication 要求 Win8，DwmSharedSurface 是未记录的 API，Win7 看来不能用。只支持 GDI 的话捕获模式选项可以删除了。

还有其他要注意的：
1. 不支持 DirectFlip 因此延迟会高一些
2. 一些图标 Win7 没有
3. 使用了很多 Win7 以后添加的 API，有些可能没有替代品
4. Win7 对高 DPI 的支持很差

虽然我不知道有多少人还在坚守 Win7，非常感谢你的探索

**Comment by HIllya51** [comment]
直接在dll的run里面让模式等于2了！偷了个懒！
1、莫得办法，win7不支持
2、emm这个影响不大
3&4、是的呢，我用一些win7支持的api进行了替换，勉强进行了支持
哈哈，谢谢🤗

**Comment by wyghthy** [comment]
非常感谢，终于有视窗七能用的了！
